src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import cn from 'classnames';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const MALE = 'm';
+  let partnerLabel;
+
+  if (isMarried) {
+    partnerLabel = sex === MALE ? 'wife' : 'husband';
+  } else {
+    partnerLabel = 'not married';
+  }
+
+  const partnerText = cn({
+    [`${partnerName} is my ${partnerLabel}`]: isMarried && partnerName,
+    [`I am ${partnerLabel}`]: !isMarried || !partnerName,
+  });
```

Comment:
Your code is okay, but I would do this for better readability
```suggestion
  const partnerLabel = sex === MALE ? 'wife' : 'husband';
  const partnerText = isMarried ? `${partnerName} is my ${partnerLabel}` : 'I am not married';
```